core_test.go

```diff
@@ -213,6 +213,99 @@ func TestMatToBytes(t *testing.T) {
 	}
 }
 
+func TestMatDataPtr(t *testing.T) {
```

Comment:
Seems to me like it would be a good idea to have code in these tests to set/get a few values in the Mat.

Reply:
Looking good now, thanks for adding the additional tests.